validator/main.go

```diff
@@ -641,69 +725,77 @@ func (d *Driver) validate() error {
 		return err
 	}
 
-	// delete host driver status file is already present
-	err = deleteStatusFile(outputDirFlag + "/" + hostDriverStatusFile)
+	driverInfo, err := d.runValidation(false)
 	if err != nil {
+		log.Error("driver is not ready")
```

Comment:
Let's log the error in this line too
```suggestion
		log.Error("driver is not ready")
```

Reply:
Done